docker-compose.yml

```diff
@@ -7,19 +7,19 @@ proxy:
    - "8888:80"
    - "5555:5555"
   volumes:
-   - proxy/nginx/conf:/opt/openresty/nginx/conf
-   - proxy/nginx/lualib:/opt/openresty/nginx/lualib
-   - proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
-   - proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
+   - ./proxy/nginx/conf:/opt/openresty/nginx/conf
+   - ./proxy/nginx/lualib:/opt/openresty/nginx/lualib
+   - ./proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
+   - ./proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
 api:
   extends:
     file: base.yml
     service: api
-  build: api
+  image: jazzdd/alpine-flask
```

Comment:
Shouldn't we use `python:alpine` as base and build the requirements in? Otherwise, the requirements are missing and this `docker-compose` file will fail I think.